.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please add `.vscode` to `.gitignore`

Reply:
Add to `.gitignore` and do `git rm` on this file